src/main/java/se/bjurr/violations/lib/example/OtherClass.java

```diff
@@ -4,11 +4,14 @@
  * No ending dot
  */
 public class OtherClass {
- public static String CoNstANT = "yes";
+ public static String CoNstANT = "yes"; 
 
  public void myMethod() {
   if (CoNstANT.equals("abc")) {
 
+  }
+  if (CoNstANT.equals("abc")) {
+
   }
  }
```

Comment:
**Reporter**: FINDBUGS
**Rule**: UC_USELESS_VOID_METHOD
**Severity**: INFO
**File**: se/bjurr/violations/lib/example/OtherClass.java L16
**Source**: se.bjurr.violations.lib.example.OtherClass

Useless non-empty void method&#10;&#10;&#10;&lt;p&gt;Our analysis shows that this non-empty void method does not actually perform any useful work.&#10;Please check it: probably there&#39;s a mistake in its code or its body can be fully removed.&#10;&lt;/p&gt;&#10;&lt;p&gt;We are trying to reduce the false positives as much as possible, but in some cases this warning might be wrong.&#10;Common false-positive cases include:&lt;/p&gt;&#10;&lt;p&gt;- The method is intended to trigger loading of some class which may have a side effect.&lt;/p&gt;&#10;&lt;p&gt;- The method is intended to implicitly throw some obscure exception.&lt;/p&gt;&#10;


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a368927103>*